docs/windows.md

```diff
@@ -130,7 +132,8 @@ kube-proxy version.
 
 ```bash
 # Example:
-curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml | sed 's/VERSION/v1.18.0/g'  > kube-proxy.yml
+$SigWindowsToolsVersion="v0.1.5"
```

Comment:
Since this is a guide document, I think you could have one generic introduction about the example version of variable ` $SigWindowsToolsVersion` and `$kubernetesVersion` we used in the examples, then we can use the variable directly anywhere. And we can explain sig-windows v0.1.5 is tested if you feel necessary.

Reply:
Used `v0.1.5` for sig-windows-tool version. We think we can update kubernetes version in another PR after we verified a newer one than `v1.18.0`.